packages/html-template/template.ejs

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="<%= htmlWebpackPlugin.options.lang %>">
+  <head>
+    <meta charset="utf-8">
```

Comment:
It's super picky, but can you make this tag self-closing? The reasoning being `xhtml` mode. If they enable xhtml mode, this would be the one place in the template that isn't compliant, while a self-closing tag will be compliant with either mode.

Reply:
Neither the stock template, nor the one we've been using until now, nor CRA's do that:
https://github.com/jantimon/html-webpack-plugin/blob/55abb36ee8cad6e8eb32fcc0848c785099316f2d/default_index.ejs#L4
https://github.com/jaketrent/html-webpack-template/blob/71257cad015f0e507da13435adef6b648d8eadc9/index.ejs#L10
https://github.com/facebook/create-react-app/blob/a671462c1089c1afe95cc1f9d4f2432d3a885806/packages/react-scripts/template/public/index.html#L4

I think XHTML compliant markup falls into "the 1%". See also a recent thread on the Django developers list about how it's not really worth achieving:
https://groups.google.com/forum/#!topic/django-developers/G4Rz1_Bmh7w